acquisitions/src/main/resources/thunderjet/mod-data-export-spring/features/edifact-orders-export.feature

```diff
@@ -0,0 +1,235 @@
+
+    And assert karate.sizeOf(jobs) == 1
+    * def job = jobs[0]
+    And assert job.status == 'FAILED'
+    And assert job.errorDetails == 'Orders for export not found (OrderNotFoundException)'
+
+  Scenario: If there is an open order for organization and organization has integration method with the same acquisition method as order but not for (today) current day of the week THEN there should not be a job for today (ediSchedulePeriod = 'WEEK')
+    # Create an organization
+    * def extOrganizationId = call uuid1
+    * def extOrganizationName = 'MODEXPS-202-OrgName-5'
+    * def extOrganizationCode = 'MODEXPS-202-OrgCode-5'
+    * call read('util/initData.feature@CreateOrganization') { extOrganizationId: #(extOrganizationId), extOrganizationName: #(extOrganizationName), extOrganizationCode: #(extOrganizationCode) }
+
+    # Create an account, and set it to the organization
+    * def extAccount = read('samples/account-for-organization.json')
+    * def extAccountName = 'MODEXPS-202-OrgAccountName-5'
+    * def extAccountNo = 'MODEXPS-202-OrgAccountNo-5'
+    * extAccount.name = extAccountName
+    * extAccount.accountNo = extAccountNo
+
+    * call read('util/initData.feature@SetAccountToOrganization') { extOrganizationId: #(extOrganizationId), extAccounts: [#(extAccount)] }
+
+    # Create purchase order for the organization
+    * def extOrderId =  call uuid1
+    * def extPoNumber =  'PoNumber5'
+    * call read('util/initData.feature@CreateOrderForOrganization') { extOrderId: #(extOrderId), extOrganizationId: #(extOrganizationId), extPoNumber: #(extPoNumber) }
+
+    # Create PO Line for the order
+    * def extPoLineId = call uuid1
+    * call read('util/initData.feature@CreateOrderLines') { extPoLineId: #(extPoLineId), extOrderId: #(extOrderId), extOrganizationId: #(extOrganizationId), extAccountNumber: #(extAccountNo) }
+
+    # Open the order
+    * call read('util/initData.feature@OpenOrder') { extOrderId: #(extOrderId)}
+
+    # Add integration to the organization
+    * def extExportConfigId = call uuid1
+
+    * def currentDay = currentDayOfWeek('UTC')
+
+    * def extEdiScheduleFrequency = 1
+    * def extEdiSchedulePeriod = 'WEEK'
+    * def extEdiScheduleTime = nextZonedTimeAsLocaleSettings('UTC', 1)
+    * def extEdiScheduleWeekDays = karate.filter(['SATURDAY', 'FRIDAY', 'THURSDAY', 'WEDNESDAY', 'TUESDAY', 'MONDAY', 'SUNDAY'], (day) => day != currentDay)
+    * def extEdiScheduleTimeZone = 'UTC'
+    * call read('util/initData.feature@AddIntegrationToOrganization') { extExportConfigId: #(extExportConfigId), extOrganizationId: #(extOrganizationId), extAccountNoList: [#(extAccountNo)], extEdiScheduleFrequency: #(extEdiScheduleFrequency), extEdiSchedulePeriod: #(extEdiSchedulePeriod), extEdiScheduleTime: #(extEdiScheduleTime), extEdiScheduleWeekDays: #(extEdiScheduleWeekDays), extEdiScheduleTimeZone: #(extEdiScheduleTimeZone)}
+
+    # Pause for a minute ('delay' = 1 minute)
+    * call pause 70000
```

Comment:
Here you setup ro run export on the next minute? so waiting for 70 seconds to be sure that max available value for minute = 60 seconds  + 10 seconds for completing of export?

Reply:
Yes. Previously it was 60 seconds, but when setting 'ediSchedulePeriod' to 'WEEK', it is taking a bit longer time so result was not consistent.

Reply:
Thank you, I found the cause of inconsistency. Now we need 60 seconds + 5 seconds AFTER the POSTing schedule, and 10 seconds BEFORE the POSTing a schedule if necessary